services/order.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str,

```

Comment:
Place each argument on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability